Makefile

```diff
@@ -42,15 +42,29 @@ LINT_EXCLUDES := $(GENERATED_GO_FILES) $(LINT_EXCLUDES_EXTRAS)
 FILTER_LINT := grep -v $(patsubst %,-e %, $(LINT_EXCLUDES))
 
 BUILD_FLAGS ?=
+RAGEL_PATH := "$(shell pwd)/vendor/ragel"
 
 .PHONY: build
 build:
 	go build -i $(BUILD_FLAGS)
 
+.PHONY: install-ragel
+install-ragel:
+	ls ./vendor/ragel 1>/dev/null 2>/dev/null || ( \
+		( curl https://www.colm.net/files/ragel/ragel-6.9.tar.gz | \
+		tar -xz ) && mv ./ragel-6.9 ./vendor/ragel \
```

Comment:
I would let `make` decide if you need to build ragel instead of having your own check to see if it exists.

```
$(RAGEL_PATH)/bin/ragel:
    mkdir -p $(RAGEL_PATH)
    curl ...
```

You can compress down the curl, tar and mv to:

```
curl ... | tar -C ./vendor/ragel -xz --strip-components 1
```

The `-C` tells it to change to this directory before extracting and the strip-components tells it to drop the first component from the path of every file in the tar -- the "ragel-6.9" directory.